### PR TITLE
Update forum link to discuss

### PR DESCRIPTION
**Pull request by tupui**
As we are transitioning, it would be good to already update this link so that it's easier on new users.

### CODE REVIEW

**Comment by tupui** [comment]
Wait to get this in, I am on the phone and I see it would be good to change some other links too.

**Comment by tupui** [comment]
Yep thanks 👍 Feel free to push on this branch @lucascolley. I will finish tonight otherwise

**Comment by lucascolley** [comment]
If we are removing the 'get help' page, do we want to add a link to the Stackoverflow tag elsewhere? Probably not needed I suppose given the amount of other channels available. @mdhaber WDYT?

**Comment by tupui** [comment]
Right, it's still in the FAQ but I can surface it in the community page 👍

File: netlify.toml [hunk 1]
```diff
@@ -44,6 +44,10 @@
   from = "/scipylib/donations.html"
   to = "/donations/"
 
+[[redirects]]
+  from = "/gethelp"
+  to = "/community/"
```
Comment (tupui) [inline netlify.toml:49]: @jarrodmillman can you confirm that I am doing this right? Thanks!

**Comment by tupui** [comment]
I did the changes @lucascolley

File: netlify.toml (hunk 1 above)
Comment (tupui) [reply netlify.toml:49]: I think this is correct, using the preview and adding `gethelp`, the redirection is going through 👍

File: content/en/community.md [hunk 2]
```diff
@@ -33,15 +37,14 @@ Once a month we have special meetings for folks who want to start contributing
 or have just started. All are welcome! Check our community calendar for details,
 or look out for the announcements on our mailing list.
 
-### [SciPy mailing list](https://mail.python.org/mailman3/lists/scipy-dev.python.org/)
+### [SciPy Forum](https://discuss.scientific-python.org/c/contributor/scipy)
 
 This list is the main forum for longer-form discussions, like adding new
```
Comment (melissawm) [inline content/en/community.md:42]: This paragraph also needs to be reworded

**Review by melissawm — COMMENTED** [review]
Thank you @tupui !

**Comment by tupui** [comment]
Thanks @melissawm for the suggestions 😃 All in!

File: content/en/community.md [hunk 3]
```diff
@@ -16,32 +16,35 @@ connect with others within the SciPy community.
...
 Once a month we have special meetings for folks who want to start contributing
 or have just started. All are welcome! Check our community calendar for details,
-or look out for the announcements on our mailing list.
+or look out for the announcements on our development forum.
 
-### [SciPy mailing list](https://mail.python.org/mailman3/lists/scipy-dev.python.org/)
+### [SciPy development forum](https://discuss.scientific-python.org/c/contributor/scipy)
 
-This list is the main forum for longer-form discussions, like adding new
+This space is the main forum for longer-form discussions, like adding new
```
Comment (melissawm) [inline content/en/community.md:42]: sorry one last "list" at the end of this paragraph 😅

**Review by melissawm — APPROVED** [review]
Looks good with a last tiny comment, thanks again!

**Comment by tupui** [comment]
lol thanks for catching that 👍

**Comment by tupui** [comment]
It's green and you approved so getting this in. Thanks again both 😃

**Review by jarrodmillman — APPROVED** [review]
LGTM once you finish responding the @melissawm feedback

**Comment by tupui** [comment]
Nice timing, thanks @jarrodmillman

**Pull request merged**